src/models/tools/geometry/geometry-content.ts

```diff
@@ -308,7 +309,7 @@ export const GeometryContentModel = types
       const linkedTiles = lastChangeLinks ? lastChangeLinks.tileIds : undefined;
       const linkedTile = linkedTiles && linkedTiles[0];
       const tableContent = linkedTile ? getTableContent(self, linkedTile) : undefined;
-      return tableContent ? tableContent.canUndoLinkedChange(lastChangeParsed) : false;
+      return tableContent ? tableContent.canUndoLinkedChange(/*lastChangeParsed*/) : false;
```

Comment:
I eliminated it from `TableContent.canUndoLinkedChange()` because it isn't using the argument and it didn't make sense to be importing `geometry-content.ts` into `table-content.ts` for a type that isn't actually used. When we actually get around to implementing undo/redo we'll have to revisit these issues.

Reply:
:+1: